recipes/fbclient/3.x.x/CMakeLists.txt

```diff
@@ -0,0 +1,280 @@
+cmake_minimum_required(VERSION 3.16)
```

Comment:
is it taken from somewhere or written from scratch?

Reply:
Refer to OP. I wrote this.